.circleci/config.yml

```diff
@@ -9,13 +9,13 @@ commands:
       - run:
           name: Install z3
           command: |
-            wget https://github.com/Z3Prover/z3/releases/download/z3-4.8.7/z3-4.8.7-x64-ubuntu-16.04.zip
-            unzip z3-4.8.7-x64-ubuntu-16.04.zip
-            rm -f z3-4.8.7-x64-ubuntu-16.04.zip
-            sudo cp z3-4.8.7-x64-ubuntu-16.04/bin/libz3.a /usr/local/lib
-            sudo cp z3-4.8.7-x64-ubuntu-16.04/bin/z3 /usr/local/bin
-            sudo cp z3-4.8.7-x64-ubuntu-16.04/include/* /usr/local/include
-            rm -rf z3-4.8.7-x64-ubuntu-16.04
+            wget https://github.com/Z3Prover/z3/releases/download/z3-4.13.3/z3-4.13.3-x64-glibc-2.35.zip
+            unzip z3-4.13.3-x64-glibc-2.35.zip
+            rm -f z3-4.13.3-x64-glibc-2.35.zip
+            sudo cp z3-4.13.3-x64-glibc-2.35/bin/libz3.a /usr/local/lib
+            sudo cp z3-4.13.3-x64-glibc-2.35/bin/z3 /usr/local/bin
+            sudo cp z3-4.13.3-x64-glibc-2.35/include/* /usr/local/include
+            rm -rf z3-4.13.3-x64-glibc-2.35
```

Comment:
We should add some check for this version of Z3. Otherwise, tests are going to fail for some users, and they will be clueless about why. [Liquid.Fixpoint.Smt.Interface](https://github.com/ucsd-progsys/liquid-fixpoint/blob/a7e09a68e90b81fbc428d2acf655b25a21298584/src/Language/Fixpoint/Smt/Interface.hs#L332) does version checks already.

Then it would be fine to skip the version check with a flag `--skip-smt-version-check` if the user wants to run an older version anyway.

Reply:
I believe that was just a z3 bug so i don't know if it makes sense signaling to the user, and if we decide to do that we should probably check exactly from which version it's not going to loop forever, on Niki's pc the version was just slightly older than the CI's one

Reply:
I'm assuming it is a z3 bug, indeed. I think you can drop this if you prefer. We can have an issue specifically for checking the z3 version.

My local version of z3 is 4.8.17.

Reply:
I think an issue would be better